### ISSUE TITLE
Tags / labels

### ISSUE THREAD

**Issue by chdesign** [dwpdigitaltech/design-examples#9]
**_Screenshot_**
Tags appearing on a to-do:
![Tags used on a to-do](https://user-images.githubusercontent.com/22620603/35731434-6dbdc20e-080d-11e8-97b3-e7e116d9076b.png)

Tags appearing on a page:
![Tags used in a page, beneath the header](https://user-images.githubusercontent.com/22620603/35731435-6dd55766-080d-11e8-83df-05c6df402eb8.png)

Option available to filter by tag:
![Tags used to filer](https://user-images.githubusercontent.com/22620603/35731497-b34d76c0-080d-11e8-97c0-78f68adc0195.png)

**_What problem does this design solve?_**
Tags are used to categories to-do's on the service, which then allows the user identify and to filter the list of to-dos.

**_Which service are you researching this design on?_**
Prepare for Universal Credit

**_How have you approached the design?_**
- have you tried anything that exists already?
_Looking into_

**_What have you learned from research?_**
- include any iterations that didn't work.
_Looking into_

**_Have you tested this for accessibility?_**
I tested for contrast and fixed those that failed. Research should be done to find out if different colours are even needed though.

**_Link to code_**
_Needs doing_

**Comment by htmlandbacon** [comment]
**_Screenshot_**

<img width="299" alt="status" src="https://user-images.githubusercontent.com/9450064/34210361-77d8be9e-e58d-11e7-93ac-1b7e998398d1.png">

**_What problem does this design solve?_**
A quick way to show agents the status of the claim they are working on.

**_Which service are you researching this design on?_**
- Bereavement Support Payment - The badges are in the production build of the payment scheduler, and we are continuing with the same format in the prototypes for the claim management.

**_How have you approached the design?_**
The design is a combination of the GOVUK .x-large heading, GOVUK subheading and the badge which was taken from [companies house](https://beta.companieshouse.gov.uk/company/10344843/officers).

**_What have you learned from research?_**
The original design was plain text but it went unnoticed by most agents we tested with. Since adding the colours the agents were able to quickly recognise the status of the claim.

**_Have you tested this for accessibility?_**
- It has been tested with users of JAWS, Dragon, Zoomtext, and Read and Write Gold. This is both agents on their work devices and the DWP accessibility compliance team.

Notes: I've moved this from another issue, after speaking to @abbott567 we agreed it felt more like a **label** than a **status badge**.

**Comment by petegriffithdwp** [comment]
![cfems - team leader](https://user-images.githubusercontent.com/35726405/36907361-5aafde7a-1e30-11e8-8b26-95c98bb517ab.png)

_**What problem does this design solve?**_
Tags are used to flag up to agents and their managers when an activity on a case is either nearing its due date or has overrun it.

_**Which service are you researching this design on?**_
Criminal Fraud and Error Management Services (CFEMS), formally known as FES Work Management.

_**How have you approached the design?**_
The original design was decided upon before my arrival on the project but I have since made changes such as having the label sit next to the due date (as it's directly related) and also linking the labels to the activity data in the top right corner (which summarises the amount of activities that are 'due soon' and/or 'overdue'). 

_**What have you learned from research?**_
Tags were intuitive to the users and allowed them to immediately spot which activities they needed to focus on when managing their work for the day. This also helped Team Leaders to understand their team's need for support and helped inform their allocation of work to individuals.

_**Have you tested this for accessibility?**_
This iteration of the design is yet to be built and will require testing once implemented.

**Comment by abbott567** [comment]
Hi @chdesign - Looking at these two examples they're pretty similar. Was there a reason you rounded the corners on your labels?

**Comment by chdesign** [comment]
I didn't design them I inherited them on Prepare for UC (PUC). Looking at the other examples, the ones on PUC were used to tag content and could be clicked on to filter, whereas the others look like a status indicator which the GDS design system already has. Maybe these were meant to be visually distinct as they behave slightly differently.

![image](https://user-images.githubusercontent.com/22620603/41290317-721260ce-6e44-11e8-9fe7-0dbc66a36dcf.png)

**Comment by abbott567** [comment]
Yeah the ones in my service were modelled off the govuk alpha banner. We do change the colours, which isn't available in the GOVUK tag component. We should probably just create modifier classes for the existing component for tags. Might be worth moving labels into it's own issue or grouping it with filtering.

**Comment by chdesign** [comment]
Done. I've moved off to filtering and made clear that the tags there are used to aid filtering, that issue can contain other examples of filtering now, which I'm sure we have too.

**Comment by abbott567** [comment]
Moved to experimental now we have code examples for tags:
https://dwp-design-examples.herokuapp.com/example/tags

**Comment by chdesign** [comment]
Outcome from the design examples meet 6 September 2018:
- use a modifier on current [tag component](https://design-system.service.gov.uk/components/tag/) – these need a coded example
- recommendation for now is to only use for statuses not for categories, eg as in the Prepare for UC service
- don’t use colour for aesthetics – more work needed on colours that might be useful, green/red for accepted/declined for example
